index.html

```diff
@@ -2,25 +2,30 @@
 <html>
 <head>
   <meta charset="utf-8" />
-  <title>Black Screen</title>
-  <link rel="stylesheet" type="text/css" href="compiled/all.css" media="screen"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title></title>
+  
+  <!-- TOOD: Merge and minimify -->
+  <link rel="stylesheet" type="text/css" href="dist/main.css" media="screen"/>
```

Comment:
Yes, but I thought in a future if we minify the app scripts could reduce a bit the size of the packaged app. Now, this app is on an early stage so I only added a comment, if you want I remove it.